app/models/Actionlog.php

```diff
@@ -1,85 +1,193 @@
 
-    public function adminlog()
-    {
-        return $this->belongsTo('User','user_id')->withTrashed();
-    }
+        public function licenselog()
+        {
 
-    public function userlog()
-    {
-        return $this->belongsTo('User','checkedout_to')->withTrashed();
-    }
+            return $this->belongsTo( 'License', 'asset_id' )
+                        ->withTrashed();
+        }
 
-    public function childlogs()
-    {
+        public function accessorylog()
+        {
 
-        return $this->hasMany( 'ActionLog', 'thread_id' );
-    }
+            return $this->belongsTo( 'Accessory', 'accessory_id' )
+                        ->withTrashed();
+        }
 
-    public function parentlog()
-    {
+        public function consumablelog()
+        {
 
-        return $this->belongsTo( 'ActionLog', 'thread_id' );
-    }
+            return $this->belongsTo( 'Consumable', 'consumable_id' )
+                        ->withTrashed();
+        }
 
-    /**
-	* Check if the file exists, and if it does, force a download
-	**/
-    public function get_src() {
+        public function adminlog()
+        {
 
-			$file = app_path().'/private_uploads/'.$this->filename;
-			return $file;
+            return $this->belongsTo( 'User', 'user_id' )
+                        ->withTrashed();
+        }
 
-    }
+        public function userlog()
+        {
 
+            return $this->belongsTo( 'User', 'checkedout_to' )
+                        ->withTrashed();
+        }
 
+        public function childlogs()
+        {
 
-    /**
-    * Get the parent category name
-    */
-    public function logaction($actiontype)
-    {
-        $this->action_type = $actiontype;
+            return $this->hasMany( 'ActionLog', 'thread_id' );
+        }
 
-        if($this->save()) {
-            return true;
-        } else {
-            return false;
+        public function parentlog()
+        {
+
+            return $this->belongsTo( 'ActionLog', 'thread_id' );
+        }
+
+        /**
+         * Check if the file exists, and if it does, force a download
+         **/
+        public function get_src()
+        {
+
+            $file = app_path() . '/private_uploads/' . $this->filename;
+
+            return $file;
+
+        }
+
+        /**
+         * Get the parent category name
+         */
+        public function logaction( $actiontype )
+        {
+
+            $this->action_type = $actiontype;
+
+            if ($this->save()) {
+                return true;
+            } else {
+                return false;
+            }
         }
-    }
-}
+
+        /**
+         * getUnacceptedAssets
+         * @param array $listOfAssets
+         *
+         * @return mixed
+         * @static
+         * @author  Vincent Sposato <vincent.sposato@gmail.com>
+         * @version v1.0
+         */
+        public static function getUnacceptedAssets( $listOfAssets = [ '*' ] )
+        {
+
+            $assetIds = implode( ",", $listOfAssets );
```

Comment:
No, I don't think so. The weird part is that it's not complaining that the column is missing, it's saying the entire table is missing. It copies the sample database.example.php file into the tests directory and uses that, and there are no table prefixes there.